src/main.py

```diff
@@ -75,3 +77,23 @@ def print_banner():
 # Step 3: Remove namespace references within schemas
 print("\n Step 3: Removing namespace references from schemas...")
 remove_namespace_references(schema_files, list(namespaces))
+
+# Step 4: Build dependency matrix and resolve transitive dependencies
+print("\n Step 4: Building and resolving dependency graph...")
+deps = build_dependency_matrix(schema_files, schema_names)
+combined_deps = resolve_dependencies(deps)
+
+# Step 5: Merge dependencies and wrap each schema in ASN.1 definition blocks
+print("\n Step 5: Merging dependencies and wrapping schema definitions...")
+merge_dependencies(schema_files, combined_deps)
+wrap_definitions(schema_files)
+
+# Step 6: Verify ASN.1 syntax and structure
+print("\n Step 6: Verifying ASN.1 syntax and structure...")
+
+# Step 7: Copy required message types
+print("\n Step 7: Copying final message types to output folder...")
+verify_syntax(schema_files)
```

Comment:
It doesn't look like `verify_syntax(...)` has a return value, and it catches the possible ParseExceptions from asn1tools, so other than the text output to a log file (see my other comment) the results of this check don't seem to be used anywhere? Should we return a pass/fail status from that function and check it here, failing gracefully if there are errors?